railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb.tt

```diff
@@ -61,7 +61,7 @@ class <%= controller_class_name %>Controller < ApplicationController
       <%- if attributes_names.empty? -%>
       params.fetch(:<%= singular_table_name %>, {})
       <%- else -%>
-      params.require(:<%= singular_table_name %>).permit(<%= attributes_names.map { |name| ":#{name}" }.join(', ') %>)
+      params.require(:<%= singular_table_name %>).permit(<%= attributes_names.map { |name| ":#{name}, :#{name}_aip" }.join(', ') %>)
```

Comment:
this is apparently only used by `rails g scaffold_controller ...` and not by the normal `scaffold`. at least these changes have no effect whatsoever

Reply:
it's not used at all. when using the normal scaffold it also logs `invoking scaffold_controller` and even when calling `scaffold_controller` directly, this code does not change anything. The generated controller all in all also looks slightly different from what should be generated according to this